src/blocks/social-profiles/index.php

```diff
@@ -160,7 +160,7 @@ function coblocks_render_social_profiles_block( $attributes ) {
 		$class .= ' has-colors';
 	}
 
-	if ( isset( $attributes['size'] ) && 'med' !== $attributes['size'] && ( isset( $attributes['className'] ) && strpos( $attributes['className'], 'is-style-mask' ) === false ) ) {
```

Comment:
For some reason, we were intended for this `has-button-size-` class logic to be excluded when we have the 
`mask` style set. 
```js
strpos( $attributes['className'], 'is-style-mask' ) === false )
```
It would be prudent to understand why this decision was made and to purposefully exclude this specific requirement if we determine the logic is not needed.